src/Microsoft.OpenApi.OData.Reader/Microsoft.OpenAPI.OData.Reader.csproj

```diff
@@ -15,14 +15,15 @@
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <PackageId>Microsoft.OpenApi.OData</PackageId>
     <SignAssembly>true</SignAssembly>
-    <Version>1.2.0-preview2</Version>
+    <Version>1.2.0</Version>
```

Comment:
let's still use a preview version (4or 5?) so we can do some quality work with one weekly generation before we release the changes.

Reply:
Going with `-preview3`